src/wp-includes/class-wp-theme-json.php

```diff
@@ -2463,7 +2504,8 @@ protected static function get_default_slugs( $data, $node_path ) {
 	 * @return string|null
 	 */
 	protected function get_name_from_defaults( $slug, $base_path ) {
-		$path            = array_merge( $base_path, array( 'default' ) );
+		$path            = $base_path;
```

Comment:
Same question about `$path = array();`

Reply:
See above, that would change the behavior.